tsparser/src/parser/usageparser/mod.rs

```diff
@@ -186,13 +186,11 @@ impl<'a> UsageResolver<'a> {
                         };
 
                         for bind in resolved_binds.into_iter().flatten() {
-                            if let Some(name) = &bind.name {
```

Comment:
I think we should map the `None` case to `"default"`, as I believe that is how JavaScript models default exports.

Reply:
:heavy_check_mark: